src/Mod/PartDesign/App/FeatureLoft.cpp

```diff
@@ -213,7 +213,23 @@ App::DocumentObjectExecReturn *Loft::execute()
             std::vector<TopoShape> backwires;
             for(auto& sectionWires : wiresections)
                 backwires.push_back(sectionWires.back());
-            back = TopoShape(0).makeElementFace(backwires);
+            const char *faceMaker[] = {
+                "Part::FaceMakerBullseye",
+                "Part::FaceMakerCheese",
+                "Part::FaceMakerSimple",
+            };
+            for (int i = 0; i < std::size(faceMaker); i++) {

```

Comment:
There's a lint warning here:
`comparison of integers of different signs: 'int' and 'size_t' (aka 'unsigned long') [-Wsign-compare]`

```suggestion
            for (size_t i = 0; i < std::size(faceMaker); i++) {
```
Do you want to fix this yourself or should I add it as a separate commit before merging?